README.md

```diff
@@ -4,7 +4,105 @@
 
 The JPO Intersection GeoJSON Converter is a real-time validator and data converter of JPO-ODE MAP and SPaT JSON based on the SAE J2735 message standard. Messages are consumed from Kafka and validated based on both the SAE J2735 standard and the more robust Connected Transportation Interoperability (CTI) Intersection Implementation Guide Message Requirements (Section 3.3.3). Message validation occurs simultaneously as the GeoJSON converter converts the JPO-ODE MAP and SPaT messages into mappable geoJSON. The JPO Intersection GeoJSON Converter outputs the resulting geoJSON onto Kafka topics. These messages contain validation information that identifies all issues encountered with validation, if any.
 
-![alt text](docs/jpo-geojsonconverter_arch_diagram.png "jpo-geojsonconverter Design Diagram")
+```mermaid
+flowchart LR
+
+%% Column 0: The Source
+subgraph COL0 ["Source"]
+    direction TB %% <-- Keep columns vertical
+    ODE["ODE (Operational Data Environment)"]
+end
+
+%% Column 1: Input Topics
+subgraph COL1 ["Kafka - Input Topics"]
+    direction TB
+    A_Map["OdeMapJson"]
+    A_Spat["OdeSpatJson"]
+    A_Bsm["OdeBsmJson"]
+    A_Psm["OdePsmJson"]
+    A_Rtcm["OdeRtcmJson"]
+    A_Srm["OdeSrmJson"]
+    A_Ssm["OdeSsmJson"]
+    A_Tim["OdeTimJson"]
+end
+
+%% Column 2: Processing
+subgraph COL2 ["Processing Service<br/>(jpo-geojsonconverter)"]
+    direction TB
+    V_Map["Validate Map"] --> C_Map["Convert Map"]
+    V_Spat["Validate Spat"] --> C_Spat["Convert Spat"]
+    V_Bsm["Validate Bsm"] --> C_Bsm["Convert Bsm"]
+    V_Psm["Validate Psm"] --> C_Psm["Convert Psm"]
+    V_Rtcm["Validate Rtcm"] --> C_Rtcm["Convert Rtcm"]
+    V_Srm["Validate Srm"] --> C_Srm["Convert Srm"]
+    V_Ssm["Validate Ssm"] --> C_Ssm["Convert Ssm"]
+    V_Tim["Validate Tim"] --> C_Tim["Convert Tim"]

```

Comment:
Bit of a nitpick - since these aren't topic names, could we capitalize the acronyms for the message types according to their standard?